pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -388,6 +391,78 @@ class OSRow extends React.Component {
     }
 }
 
+const UnattendedRow = ({ validationFailed, unattendedDisabled, unattendedInstallation, os, profile, onValueChanged }) => {
+    const validationStatePassword = validationFailed.password ? 'error' : undefined;
+    let unattendedInstallationCheckbox = (
+        <label className="checkbox-inline">
+            <input id="unattended-installation" type="checkbox"
+                checked={unattendedInstallation}
+                disabled={unattendedDisabled}
+                onChange={e => onValueChanged('unattendedInstallation', e.target.checked)} />
+            {_("Run unattended installation")}
+        </label>
+    );
+    if (unattendedDisabled) {
+        unattendedInstallationCheckbox = (
+            <OverlayTrigger overlay={ <Tooltip id='os-unattended-installation-tooltip'>{ _("The selected Operating System does not support unattended installations") }</Tooltip> } placement='left'>
+                {unattendedInstallationCheckbox}
+            </OverlayTrigger>
+        );
+    }
+
+    return (
+        <>
+            {unattendedInstallationCheckbox}
+            {!unattendedDisabled && unattendedInstallation ? <>
+                {os.profiles.length > 0 && <>
```

Comment:
Just to confirm that this is intended: This will show the selector even if there is just a single profile. So that's mostly just for information, not an actual control. (Fine for me, but just in case you wanted this to be `> 1`).

Reply:
Yes that's intended - I think it's important that the user is aware of the profile variant of the OS they will get.